tests/installation/await_install.pm

```diff
@@ -123,6 +123,10 @@ sub run {
     if (check_var('ARCH', 's390x')) {
         push(@tags, 'additional-packages');
     }
+    # For poo#64228, we need ensure the timeout value less than the MAX_JOB_TIME
+    my $max_job_time_bound = get_var('MAX_JOB_TIME', 7200) - 1000;
+    record_info("Computed timeout '$timeout' exceeds max_job_time_bound '$max_job_time_bound'", "consider decreasing '$timeout' or increasing 'MAX_JOB_TIME'") if $timeout > $max_job_time_bound;
```

Comment:
this can look a bit confusing when one only reads the text after clicking but not the hover test e.g. on http://openqa.nue.suse.com/tests/3980008#step/await_install/1 . I suggest to keep the first string really short, maybe "timeout exceeded", and put the other text all in the second string.

Reply:
Ok, updated code. Just wait the verify log: http://openqa.nue.suse.com/tests/3987601

Reply:
looks good. It reports "consider decreasing '22000'" where it's not clear to the user which variable to change but I guess it's ok for now.

Reply:
The verify log already done.